lib/cors-anywhere.js

```diff
@@ -200,7 +200,8 @@ var getHandler = exports.getHandler = function(options, proxy) {
     originBlacklist: [],      // Requests from these origins will be blocked.
     originWhitelist: [],      // If non-empty, requests not from an origin in this list will be blocked.
     requireHeader: null,      // Require a header to be set?
-    removeHeaders: []         // Strip these request headers
+    removeHeaders: [],         // Strip these request headers
+    addHeaders: []
```

Comment:
Rename to `setHeaders` and add a very short comment similar to the previous ones, to explain the purpose of this thing. E.g. `// Set these request headers.`

It should be renamed to `setHeaders` because the configuration property is not adding headers (i.e. keeping existing values if existent), but replacing them.